MyMath.java

```diff
@@ -0,0 +1,22 @@
+public class MyMath {
+    private static final double Pi=3.14;
+    private static final double G=9.8;
+    private static final int Two=2;
+
+    public static void subtract(){

```

Comment:
it'd better have static methods with params instead of using unuseful constants only
'''
int sum(int a, int b){
return a+b
}
'''